plugin/src/generators/code-checks/config.ts

```diff
@@ -8,12 +8,12 @@ export default {
     baseUrl: '.',
     rootDir: '.',
     sourceMap: true,
-    declaration: false,
+    declaration: true,
+    declarationMap: true,
     emitDecoratorMetadata: true,
     experimentalDecorators: true,
     jsx: 'react-native',
     allowJs: true,
-    noEmit: true,
```

Comment:
We do we need this change?
This will generate unwanted .js files when we run typescript checks using `npx tsc`